README.md

```diff
@@ -1,68 +1,45 @@
 
-## Training
-  * STEP 0. Download [LJ Speech Dataset](https://keithito.com/LJ-Speech-Dataset/) or prepare your own data.
-  * STEP 1. Adjust hyper parameters in `hyperparams.py`. (If you want to do preprocessing, set prepro True`.
-  * STEP 2. Run `python train.py 1` for training Text2Mel. (If you set prepro True, run python prepro.py first)
-  * STEP 3. Run `python train.py 2` for training SSRN.
+## The Task
 
-You can do STEP 2 and 3 at the same time, if you have more than one gpu card.
+For our task we did not just want to replicate the original results by using the pre-defined sentences that are synthesized, but we also wanted to be able to dynamically run inference on the Text-to-Speech model. This means that, besides replicating the previous synthesis, we wanted to be able to either use the examples from the Harvard sentences or to use our own sentences to synthesize. In terms of results, we expect the TTS synthesizer to be able to work with any input from the English language.
 
-## Training Curves
+## How does the code work?
 
-<img src="fig/training_curves.png">
+Here, you have two choices: 
+ * Simply press  [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1lg4HmCD_GhuoJjLPpfel0npiw5FxBQxk?authuser=1#scrollTo=ZNFKOgHQOtGL)  to open our Google Colab notebook. 
+ * Or: download the [DC_TTS_test.ipynb](https://github.com/jkuhlemann/dc_tts/blob/test_inference/DC_TTS_test.ipynb) to your computer and then open it in a Google Colaboratory environment and follow each step. 
 
-## Attention Plot
-<img src="fig/attention.gif">
+The Python notebook should be self-explanatory and easy to follow. Basically, it follows these steps in order to test the TTS model:
 
-## Sample Synthesis
-I generate speech samples based on [Harvard Sentences](http://www.cs.columbia.edu/~hgs/audio/harvard.html) as the original paper does. It is already included in the repo.
+  * STEP1: Clone the repository
+  * STEP2: Create necessary folders
+  * STEP3: Download & extract the pre-trained model
+  * STEP4: Customize sentences to be used for TTS synthesis or stick with pre-defined ones
+  * STEP5: Synthesize the sentences and play the desired audio-file
 
-  * Run `synthesize.py` and check the files in `samples`.
+**If you run into issues at the point of synthesizing the speech, you probably forgot to turn on GPU hardware acceleration within your Google Colab session.** 
+It is also crucial to switch to Tensorflow 1.x (already included in the notebook).

```

Comment:
There are also other complications that may occur during synthesizing that don't have to do with the GPU hardware acceleration. Perhaps be clearer and say that this could be one of many problems? 

Also: We ran into the same Tensorflow problem, great you picked it up in the readme!

Reply:
added your suggestion!